src/main/scala/beam/sim/BeamServices.scala

```diff
@@ -112,9 +115,16 @@ object BeamServices {
     new TAZTreeMap(tazQuadTree)
   }
 
-  def getTazTreeMap(file: String): TAZTreeMap = {
-    Try(TAZTreeMap.fromCsv(file)).getOrElse {
-      BeamServices.defaultTazTreeMap
+  def getTazTreeMap(filePath: String): TAZTreeMap = {
+    try {
+      TAZTreeMap.fromCsv(filePath)
+    } catch {
+      case fe: FileNotFoundException =>
+        logger.error("No TAZ file found at given path : " + filePath,fe)
```

Comment:
Could you use the format version instead of string concatenation.

Reply:
done @JustinPihony . Can you please review again.